socrata/revisions.py

```diff
@@ -243,7 +243,8 @@ def source_as_blob(self, filename, parse_options = {}):
         """
         Create a source from a file that should remain unparsed
         """
-        return self.create_upload(filename, parse_options.update({'parse_source': False}))
```

Comment:
ah so this was potentially overriding passed in options. ok

Reply:
no :( the method doesn't return anything, just mutates, so I was passing None here.

Reply:
ah man tough to bounce from elixir to imperative-style lang

Reply:
why doesn't it work like that other method I made up....